packages/okta-angular/test/e2e/harness/scripts/prebuild.js

```diff
@@ -30,8 +28,7 @@ const defaultEnvValues = {
   BASE_URI: 'http://localhost:3000',
   REDIRECT_URI: 'http://localhost:3000/implicit/callback',
   ISSUER: 'https://{yourOktaDomain}.com/oauth2/default',
-  CLIENT_ID: '{clientId}',
-  CLIENT_SECRET: '{clientSecret}',
```

Comment:
I think we're going to need to remove this from `src/environments/environments.ts.environments` so our `prebuild.js` script doesn't error looking for it in the template.